core/drivers/fabric-driver/server/events.ts

```diff
@@ -323,6 +323,32 @@ async function lookupEventSubscriptions(
     }
 }
 
+async function readAllEventMatchers(): Promise<Array<eventsPb.EventMatcher>> {
+    var returnMatchers = []
+    let db: DBConnector;
+    console.debug(`start readAllEventMatchers()`);
+    try {
+        // Create connection to a database
+        db = new LevelDBConnector(DB_NAME!);
```

Comment:
Why don't you pass this as a function parameter, so that this function can be reused to read different databases on demand?

Reply:
Currently I did the way it was done in other functions.
I think the idea was database can be configured (type and path) when driver is started based on env variables. After that all functions would refer to same database.

Reply:
I get your point. Database is assumed to be Level DB everywhere, right? I guess that's something we can abstract away later then.